SQRLDotNetClientUI/Assets/Localization/localization.json

```diff
@@ -164,7 +164,14 @@
     { "RadButExortWithRescueCode": "Require the super-secure Rescue Code for later decryption and use of the backup for maximum storage security." },
     { "MissingLibGdiPlusErrorMessage": "It seems that you are missing the \"libgdiplus\" library. Some parts of the app will not function correctly until this library is installed.\r\n\r\nYou will find install instructions on the project's Github Wiki." },
     { "UninstallMenuItemHeader": "Uninstall this app" },
-    { "ReallyUninstallQuestion":  "Are you really sure you want to uninstall the SQRL client application from your system?"}
+    { "SwitchDBMenuItemHeader": "Switch Database" },
+    { "ReallyUninstallQuestion": "Are you really sure you want to uninstall the SQRL client application from your system?" },
+    { "DbMoveQuestion": "Currently loadded DB File Is:{3}{3}{0}{3}{3}You have chosen to load a new Db File:{3}{3}{1}{3}{3}Would you like to move this new Db File to the default SQRL Path at:{3}{3}{2}{3}{3} and then load it.{3}Or should we just load this new Db in place?" },
```

Comment:
I did enable having `\r\n` in the localization strings a while ago Jose, you don't need to pass newlines in using parameters anymore.

Reply:
ah cool so do I just hard code \r\n in the string? @alexhauser ?

Reply:
Got it!